### PR TITLE
Fix/pr5

### PR DESCRIPTION
**Pull request by tomap**
To solve issues on PR #5

### CODE REVIEW

**Comment by JLHwung** [comment]
Could you please run `git rebase -i origin/master` to squash the commits with similar title?

**Comment by tomap** [comment]
@JLHwung  I was hoping to squash all commits in a single commit (from GitHub => "Squash and merge"), to hide the mess I did while pushing all those fixes

File: README.md [hunk 1]
```diff
@@ -14,13 +14,15 @@ $ npm install hexo-generator-archive --save
...
 archive_generator:
+  enabled: true
   per_page: 10
   yearly: true
   monthly: true
   daily: false
   order_by: -date
 ```
 
+- **enabled**: Is hexo-generator-archive enabled (empty/null = enabled)
```
Comment (JLHwung) [inline README.md]: Let's specify that
> The default value is `true`, set to `false` if you do not want to enable that.

I don't think, though it supports, it is a good idea to mention that `empty` or `null` will be converted to `true`. The configuration should be clear and precise.

Comment (tomap) [reply README.md]: fixed

**Pull request merged**